controllers/numaresourcesoperator_controller_test.go

```diff
@@ -933,6 +933,7 @@ func reconcileObjects(nro *nropv1.NUMAResourcesOperator, mcp *machineconfigv1.Ma
 	return reconciler
 }
 
+//nolint:unused
 func nodeGroupConfigToString(conf nropv1.NodeGroupConfig) string {
```

Comment:
Why are we choosing to keep this rather than removing it? I would incline towards removing unused functions. We can always add them if needed in the future.

Reply:
you're right